mm2src/coins_activation/src/prelude.rs

```diff
@@ -1,14 +1,23 @@
+use coins::utxo::UtxoActivationParams;
 use coins::{coin_conf, CoinBalance, CoinProtocol, MmCoinEnum};
 use common::mm_ctx::MmArc;
 use common::mm_error::prelude::*;
 use serde_derive::Serialize;
 use serde_json::{self as json, Value as Json};
 use std::collections::HashMap;
 
+pub trait GetCurrentBlock {
```

Comment:
Looks like this trait name points only one functionality. Are we sure that there will be no extra function definition inside of this trait? If not, then maybe `CurrentBlock` or `CurrentBlockFunctionality` can be better naming for this trait.